src/Compilers/VisualBasic/Portable/Syntax/LambdaUtilities.vb

```diff
@@ -677,6 +677,13 @@ Namespace Microsoft.CodeAnalysis.VisualBasic
                     ' lambda expression body closure
                     Return True
 
+                Case SyntaxKind.ClassBlock, SyntaxKind.StructureBlock
```

Comment:
Modules can have synthesized constructors that initialize fields,, so yes. This means writing another test...